mrp_lot_number_propagation/views/mrp_production.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 Camptocamp SA
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+  <record id="mrp_production_form_view" model="ir.ui.view">
+    <field name="name">mrp.production.form.inherit</field>
+    <field name="model">mrp.production</field>
+    <field name="inherit_id" ref="mrp.mrp_production_form_view" />
+    <field name="arch" type="xml">
+      <!-- Place new fields in the first group while being compatible with OE  -->
```

Comment:
Can you please also set the `lot_producing_id` and the button next to it as invisible if `is_lot_number_propagated = True` and `state 'not in' (done,to_close)`?

Reply:
There is already some `attrs` on these elements and didn't want to break the view by copying it, that's why I put the check in the `write` method to be sure this field won't be updated by mistake.
But if you think we can replace the whole `attrs` value I can do the change.

Reply:
Done through `fields_view_get` to not break the inheritance.